bindings/Sofa/src/SofaPython3/Sofa/Core/Binding_Simulation.cpp

```diff
@@ -1,57 +0,0 @@
-/*********************************************************************
```

Comment:
You removed it so it is only in SofaRuntime ?

Reply:
This was duplicated in the `Sofa.Simulation` module, the only difference being that `Sofa.Simulation` has more feature than this one. So I removed this one. Note that the binding in this class was not the binding of the Simulation class, but the binding of a python module named Simulation added to Sofa.Core.